go.mod

```diff
@@ -33,28 +33,38 @@ require (
 	github.com/vishvananda/netlink v1.1.1-0.20211118161826-650dca95af54
 	github.com/vtolstov/go-ioctl v0.0.0-20151206205506-6be9cced4810
 	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
-	golang.org/x/sys v0.0.0-20210820121016-41cdb8703e55
-	golang.org/x/term v0.0.0-20210317153231-de623e64d2a6
+	golang.org/x/sys v0.0.0-20210925032602-92d5a993a665
+	golang.org/x/term v0.0.0-20210916214954-140adaaadfaf
 	golang.org/x/text v0.3.3
 	golang.org/x/tools v0.1.1
 	gopkg.in/yaml.v2 v2.2.8
 	pack.ag/tftp v1.0.1-0.20181129014014-07909dfbde3c
 	src.elv.sh v0.16.3
 )
 
+require (
+	github.com/c-bata/go-prompt v0.2.6
+	mvdan.cc/sh/v3 v3.4.1
```

Comment:
can you merge all 3 require blocks? i'm not even sure why they're separate

Reply:
go mod tidy made them seperate, I also wondered why but as I wasn't creating them manually I just left them as is. I'll manually change it back then.

Reply:
As with gofmt, I'm more inclined to let go mod tidy guide formatting?